brainscore_language/models/huggingface.py

```diff
@@ -55,9 +56,13 @@ def digest_text(self, text: Union[str, List[str]]) -> Dict[str, DataAssembly]:
         """
         :param text: the text to be used for inference e.g. "the quick brown fox"
         :return: assembly of either behavioral output or internal neural representations
+        TODO: every function call, context/state of model is reset
         """
+        if type(text) == list:
```

Comment:
```suggestion
        if not isinstance(text, StringTypes):
```
also add `from types import StringTypes`.

This way, if the input is anything but a string, we'll try to iterate over it (including `list`, `tuple`, `np.ndarray`, etc)